BE/OwnershipAndControl/Executives.rdf

```diff
@@ -256,171 +307,93 @@
 		<rdfs:label>chief financial officer</rdfs:label>
-		<skos:definition>Senior-most executive responsible for financial control and planning of a firm or project. He or she is in charge of all accounting functions including (1) credit control, (2) preparing budgets and financial statements, (3) coordinating financing and fund raising, (4) monitoring expenditure and liquidity, (5) managing investment and taxation issues, (6) reporting financial performance to the board, and (7) providing timely financial data to the CEO. Also called chief finance officer, comptroller, controller, or finance controller.</skos:definition>
+		<skos:definition>senior-most corporate officer responsible for financial control and planning of organization or project</skos:definition>
 		<fibo-fnd-utl-av:abbreviation>CFO</fibo-fnd-utl-av:abbreviation>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/chief-financial-officer-CFO.html</fibo-fnd-utl-av:definitionOrigin>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-be-oac-exec;CompanyBylaw">
-		<rdfs:subClassOf rdf:resource="&fibo-be-oac-exec;Bylaw"/>
-		<rdfs:label>company bylaw</rdfs:label>
-		<skos:definition>The official rules and regulations which govern a corporation&apos;s management. Drawn up at the time of incorporation, along with the charter.</skos:definition>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.investorwords.com/660/bylaws.html</fibo-fnd-utl-av:definitionOrigin>
-		<fibo-fnd-utl-av:synonym>corporate bylaw</fibo-fnd-utl-av:synonym>
+		<fibo-fnd-utl-av:adaptedFrom rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/chief-financial-officer-CFO.html</fibo-fnd-utl-av:adaptedFrom>
+		<fibo-fnd-utl-av:explanatoryNote>He or she is in charge of all accounting functions including (1) credit control, (2) preparing budgets and financial statements, (3) coordinating financing and fund raising, (4) monitoring expenditure and liquidity, (5) managing investment and taxation issues, (6) reporting financial performance to the board, and (7) providing timely financial data to the CEO. Also called chief finance officer, comptroller, controller, or finance controller.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
 	<owl:Class rdf:about="&fibo-be-oac-exec;CompanyLaw">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-law-jur;StatuteLaw"/>
 		<rdfs:label>company law</rdfs:label>
-		<skos:definition>Legislation under which the formation, registration or incorporation, governance, and dissolution of a firm is administered and controlled.</skos:definition>
-		<fibo-fnd-utl-av:definitionOrigin rdf:datatype="&xsd;anyURI">http://www.businessdictionary.com/definition/company-law.html</fibo-fnd-utl-av:definitionOrigin>
+		<skos:definition>legislation under which the formation, registration or incorporation, governance, and dissolution of a firm is administered and controlled</skos:definition>
```

Comment:
Would an instance represent one specific law, or the corpus of laws (for a specific jurisdiction?).
To help answer, What is the use case for this? I cannot see any purpose for the former, only specific laws. But don't think we have sufficient to support meaningful use cases.

Reply:
I thought about deleting this ... in the US the term is corporate law, which refers both to the body of laws and the practice area.  Let's leave it for now and address it in a separate issue, as needed.  I added the synonym in an earlier change to help clarify, because the term 'company law' is the UK term, but didn't remove it.